AirCasting/SessionsSynchronization/Controller/Utils/BodyEncodingError.swift

```diff
@@ -0,0 +1,9 @@
+// Created by Lunar on 21/06/2021.
+//
+
+import Foundation
+
+/// A common `Error` type for session synchronization service adapters
+enum BodyEncodingError: Error {
+    case dataCannotBeStringified
```

Comment:
`struct BodyEncodingError: Error {}`

Reply:
But it seams that with this we are missing the underlying errors that can give us lots of information what has happened exactly.

Reply:
Please pass it as a associated value or a param